Makefile

```diff
@@ -84,7 +86,9 @@ endif
 ifneq ($(ARCH),ppc64le)
   ifneq ($(ARCH_TAG),arm32)
     ifneq ($(ARCH_TAG),riscv64)
-      CXXFLAGS += -momit-leaf-frame-pointer
+      ifneq ($(ARCH_TAG),loongarch64)
+        CXXFLAGS += -momit-leaf-frame-pointer
+      endif
```

Comment:
There are already too many nested options here - I think it's better to rewrite this with `$(findstring)` function.

Reply:
Done.